src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -1,5 +1,7 @@
 import { useToast } from '@chakra-ui/react'
-import { utxoAccountParams } from '@shapeshiftoss/chain-adapters'
+import {utxoAccountParams } from '@shapeshiftoss/chain-adapters'
+import { ChainTypes } from '@shapeshiftoss/types'
+import { FeeData } from '@shapeshiftoss/types/dist/chain-adapters'
```

Comment:
we should not be importing from `dist` directly

Reply:
good catch

Reply:
done